homeassistant/components/thermostat/homematic.py

```diff
@@ -108,21 +144,48 @@ def device_state_attributes(self):
                 "battery": self._battery,
                 "mode": self._mode}
 
+    @property
+    def min_temp(self):
```

Comment:
Default min temp for a thermostat is 7 degrees, which is too high to shut off the thermostat with an input of 4.5 degrees.